### PR TITLE
Add improvements to amp-live-list

### PR DESCRIPTION
**Pull request by kul3r4**
- Resize images
- Add margin to the UPDATE button
- Avoid dislaying a score bigger than 10
- Add draft

### CODE REVIEW

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Thanks for renaming the images. Have you checked other examples whether they use any of the renamed images?

**Comment by sebastianbenz** [comment]
Can you please mark live-list as draft.

File: src/20_Components/amp-live-list.html [hunk 1]
```diff
@@ -67,6 +60,9 @@
       padding-top: 8px;
       padding-bottom: 8px;
     }
+    .button-margin {
```
Comment (sebastianbenz) [inline src/20_Components/amp-live-list.html]: Can you find a better name for this rule? It's better to use semantic concepts for naming, e.g.: `button-update`.

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: amplivelist/amp-live-list.go [hunk 2]
```diff
@@ -51,16 +51,16 @@ var blogs []BlogItem
...
+		createBlogEntry("A green landscape", "/img/landscape_hills.jpg"),
+		createBlogEntry("A mountain landscape", "/img/landscape_mountains.jpg"),
+		createBlogEntry("A lake landscape", "/img/landscape_lake.jpg"),
+		createBlogEntry("A trees landscape", "/img/landscape_trees.jpg"),
+		createBlogEntry("A village landscape", "/img/landscape_village.jpg"),
+		createBlogEntry("A canyon", "/img/landscape_canyon.jpg"),
+		createBlogEntry("A beach", "/img/landscape_beach.jpg"),
+		createBlogEntry("Houses on the street", "/img/landscape_houses.jpg"),
+		createBlogEntry("Blue sea", "/img/landscape_sea.jpg"),
+		createBlogEntry("A ship", "/img/landscape_ship.jpg"))
```
Comment (sebastianbenz) [inline amplivelist/amp-live-list.go]: nit: can you make the labels and image names a bit more accurate?

* beach -> desert (I'm pretty sure ;-) )
* A mountain landscape -> Mountains
* A village landscape -> Scattered houses
* A ship -> A sailing ship
* A lake landscape -> Road leading to a lake

**Comment by sebastianbenz** [comment]
Could you create smaller versions of the images (width=300px) the other one are quite big (and slow). Could you then also encode the size in the file name? That'd be brilliant!

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
why did you remove the large versions?

**Comment by sebastianbenz** [comment]
The other samples don't work with small images.

**Comment by kul3r4** [comment]
Sorry, I misunderstood the previous comment then. Will put it back so that the other example will work.

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
LGTM Thanks!!!

**Pull request merged**